Update `README.md`. (#152)
yiisoft/db-oracle: README.md

    <a href="https://www.oracle.com/database/technologies/" target="_blank">
        <img src="https://avatars3.githubusercontent.com/u/4430336" height="100px">
    </a>
    <h1 align="center">Yii Database Oracle Extension</h1>
    <br>
</p>

Yii Database Oracle Extension is a database driver for [Oracle] databases that is part of the [Yii Framework]. The Yii framework is an open-source PHP framework for web application development.

> Yii Database Oracle Extension allows you to connect to [Oracle] databases from your Yii application and perform various database operations such as executing queries, creating and modifying database schema, and processing data. It supports a wide range of [Oracle] versions and provides a simple and efficient interface for working with [Oracle] databases in your Yii application.

Reviewer: ```suggestion
The package allows you to connect to [Oracle] databases from your Yii application and perform various database operations such as executing queries, creating and modifying database schema, and processing data. It supports a wide range of [Oracle] versions and provides a simple and efficient interface for working with [Oracle] databases in your Yii application.
```

[![StyleCI](https://github.styleci.io/repos/114756574/shield?branch=master)](https://github.styleci.io/repos/114756574?branch=master)

### Support version

|  PHP | Oracle Version           |  CI-Actions
|:----:|:------------------------:|:---:|
|**8.0 - 8.2**| **11 - 21**|[![build](https://github.com/yiisoft/db-oracle/actions/workflows/build.yml/badge.svg?branch=dev)](https://github.com/yiisoft/db-oracle/actions/workflows/build.yml) [![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fyiisoft%2Fdb-oracle%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/db-oracle/master) [![static analysis](https://github.com/yiisoft/db-oracle/actions/workflows/static.yml/badge.svg?branch=dev)](https://github.com/yiisoft/db-oracle/actions/workflows/static.yml) [![type-coverage](https://shepherd.dev/github/yiisoft/db-oracle/coverage.svg)](https://shepherd.dev/github/yiisoft/db-oracle)

### Installation

The package could be installed via composer:

```php
composer require yiisoft/db-oracle
```

### Config with [Yii Framework]

> The configuration with [container di](https://github.com/yiisoft/di) of [YiiFramework].

Reviewer: ```suggestion
The configuration with [DI container](https://github.com/yiisoft/di) of [YiiFramework].
```

> Also you can use any container di which implements [PSR-11](https://www.php-fig.org/psr/psr-11/).

Reviewer: ```suggestion
Also you can use any DI container which implements [PSR-11](https://www.php-fig.org/psr/psr-11/).
```
